### PR TITLE
[#376] feat: 購票說明頁面

### PR DESCRIPTION
**Pull request by yungshenglu**
## Types of changes
* **New feature**

## Description
- 購票說明頁面

## Steps to Test This Pull Request
1. Path: `/2023/zh-hant/registration/tickets` & `2023/en-us/registration/tickets`
2. Check the description of "購票說明 / Tickets" both zh-hant and English.

## Screenshots

https://user-images.githubusercontent.com/11289171/227925724-26cb305b-99d3-44af-9a5f-cc0b1c7a79e0.mov

## Related Issue
#376 

## Additional context
- Spec: https://docs.google.com/document/d/12VtFMOZD3DNMxGmhPXM24ZYxhGdK1fELNuUNGouCGNw/edit?usp=sharing
- Design: https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=5339-24303&t=LRYq1oyPspEVqoHj-0

### CODE REVIEW

File: pages/registration/tickets.vue [hunk 1]
```diff
@@ -3,23 +3,22 @@
...
-                    v-if="!shouldBreak"
-                    :title="$t('title')"
-                    class="mx-4"
-                ></core-h1>
-                <core-h1
-                    v-if="shouldBreak"
                     :title="$t('titlePart1')"
-                    class="mx-4"
+                    :show-margin-bottom="false"
+                    class="mx-4 text-[40px]"
```
[the remark this answers is not in the document]
Comment (yungshenglu) [reply pages/registration/tickets.vue:8]: Thanks for your suggestion. I've already updated, please check.

File: i18n/registration/tickets.i18n.js
Comment (josix) [inline i18n/registration/tickets.i18n.js:1]: **praise:** Nice! The structure would make the entity more clear!

File: pages/registration/tickets.vue [hunk 2]
```diff
@@ -29,95 +28,262 @@
...
+                            )"
+                            :key="`${string}.${index}`"
+                            class="leading-[30px]"
+                            :path="`beforeBuyingTickets.content.${i}.description.${index}`"
+                            tag="div"
+                        >
+                            <template #financialAid>
+                                <locale-link underline highlight>{{
+                                    $t('terms.financialAid')
+                                }}</locale-link>
```
Comment (josix) [inline pages/registration/tickets.vue:172]: **issue (blocking):** 
```suggestion
                               <locale-link
                                    :to="pageLinks.financialAid"
                                    underline
                                    highlight
                                    >{{ $t('terms.financialAid') }}
                                </locale-link>
```

File: pages/registration/tickets.vue [hunk 3]
```diff
@@ -29,95 +28,262 @@
...
+                <h2 class="text-[32px]">{{ $t('ticketsIntro.title') }}</h2>
+                <i18n
+                    path="ticketsIntro.description"
+                    tag="p"
+                    class="leading-[30px] text-[18px]"
+                >
+                    <template #financialAid>
+                        <locale-link underline highlight>{{
+                            $t('terms.financialAid')
+                        }}</locale-link>
```
Comment (josix) [inline pages/registration/tickets.vue:60]: **issue (blocking):** 
```suggestion
                               <locale-link
                                    :to="pageLinks.financialAid"
                                    underline
                                    highlight
                                    >{{ $t('terms.financialAid') }}
                                </locale-link>
```

**Review by josix — CHANGES_REQUESTED** [review]

File: i18n/registration/tickets.i18n.js [hunk 4]
```diff
@@ -2,119 +2,286 @@ import { genI18nMessages } from '@/utils/i18n.utils'
...
+        ticketsIntro: {
+            title: '購票方案',
+            description:
+                '從早鳥票到一般票，企業票至個人贊助票，您可以依照您的需求選擇您需要的票種，若需要財務補助的你，請至 {financialAid} 進行申請。',
+        },
+        beforeBuyingTickets: {
+            title: '購票前請留意',
+            content: [
+                {
+                    title: '您有需要財務補助嗎？ (尚未開放，敬請期待)',
```
Comment (josix) [inline i18n/registration/tickets.i18n.js:165]: **issue (blocking):** 
```suggestion
                    title: '您有需要財務補助嗎？',
```

File: i18n/registration/tickets.i18n.js [hunk 5]
```diff
@@ -2,119 +2,286 @@ import { genI18nMessages } from '@/utils/i18n.utils'
...
+        ticketsIntro: {
+            title: 'Tickets',
+            description:
+                'Register now and take the chance to get a 35% off discount with our Super Early Bird prices. The fee you spend will be put straight back into the community to support our diversity initiatives and financial aid programs. If you need financial aid, please visit {financialAid}.',
+        },
+        beforeBuyingTickets: {
+            title: 'Before Buying Tickets',
+            content: [
+                {
+                    title: 'Need Financial Aid? (Coming Soon.)',
```
Comment (josix) [inline i18n/registration/tickets.i18n.js:23]: **issue (blocking):** 
```suggestion
                    title: 'Need Financial Aid?',
```

**Review by josix — CHANGES_REQUESTED** [review]

File: i18n/registration/tickets.i18n.js (hunk 4 above)
Comment (yungshenglu) [reply i18n/registration/tickets.i18n.js:165]: Fixed.

File: i18n/registration/tickets.i18n.js (hunk 5 above)
Comment (yungshenglu) [reply i18n/registration/tickets.i18n.js:23]: Fixed.

File: pages/registration/tickets.vue (hunk 3 above)
Comment (yungshenglu) [reply pages/registration/tickets.vue:60]: Fixed.

File: pages/registration/tickets.vue (hunk 2 above)
Comment (yungshenglu) [reply pages/registration/tickets.vue:172]: Fixed.

**Review by josix — APPROVED** [review]
LGTM!

**Pull request merged**